tests/nosetests/pyanaconda_tests/modules/network/module_network_test.py

```diff
@@ -597,6 +598,35 @@ def teamd_requirements_test(self):
             }
         ])
 
+    @patch("pyanaconda.modules.network.network.kernel_arguments")
+    def biosdevname_requirements_test(self, mocked_kernel_arguments):
+        """Test that biosdevevname boot option results in proper requirement."""
+
+        kernel_args = KernelArguments.from_string("biosdevname=1")
+        mocked_kernel_arguments.get = kernel_args.get
```

Comment:
Btw. you can use `with` to avoid the workaround with a mocked `get` method:

`with patch("pyanaconda.modules.network.network.kernel_arguments", kernel_args):`

However, the current solution is fine.

Reply:
Ha, this is the approach I was looking for, thank you!

Reply:
Updated